src/main/java/com/gluonhq/substrate/util/FileOps.java

```diff
@@ -98,7 +98,7 @@ public static void rmdir(Path path) throws IOException {
     public static Path copyResource(String resource, Path destination) throws IOException {
         InputStream is = resourceAsStream(resource);
         if (is == null) {
-            throw new IOException("Could not copy resource named "+resource+" as it doesn't exist");
+            throw new IOException("Could not copy resource named " + resource + ", as it doesn't exist");
```

Comment:
This is much better, if we don't have the resource, we have to fail.

Reply:
ok